import-service/variables.tf

```diff
@@ -47,17 +47,57 @@ variable "service" {
 
 # sourced from https://docs.google.com/document/d/1AzTX93P35r2alE4-pviPWf-1LRBWVAF-BwrYKVVpzWo/edit
 variable "broad_range_cidrs" {
-  type    = list(string)
-  default = [ "69.173.64.0/19",
-    "69.173.96.0/20",
-    "69.173.112.0/21",
+  type = list(string)
+  default = ["69.173.112.0/21",
+    "69.173.127.232/29",
+    "69.173.127.128/26",
+    "69.173.127.0/25",
+    "69.173.127.240/28",
+    "69.173.127.224/30",
+    "69.173.127.230/31",
     "69.173.120.0/22",
-    "69.173.124.0/23",
+    "69.173.127.228/32",
     "69.173.126.0/24",
-    "69.173.127.0/25",
-    "69.173.127.128/26",
+    "69.173.96.0/20",
+    "69.173.64.0/19",
     "69.173.127.192/27",
-    "69.173.127.240/28" ]
+  "69.173.124.0/23"]
```

Comment:
assuming these changes are all intentional!

Reply:
yes this is making sure to whitelist the same [cidr blocks](https://github.com/broadinstitute/import_service_terraform/blob/8bbf31cfd1ea348057f4fc838a4b2d25fbfc59eb/import-service/setup_gae_firewall.py#L7) as the python script. I missed a couple when I first copied them

Reply:
just making sure the broad office ips match what is defined in the [infosec docs](https://dsp-security.broadinstitute.org/platform-security-categories/google-cloud-platform/securing-the-network)